### PR TITLE
Create a `repository` directive

### PR DESCRIPTION
**Pull request by ennru**
Should make it easier to consistently show non-standard repositories

- https://maven.apache.org/guides/mini/guide-multiple-repositories.html#repository-order
- https://docs.gradle.org/current/userguide/declaring_repositories.html
- https://www.scala-sbt.org/1.x/docs/Resolvers.html

### CODE REVIEW

File: docs/src/main/paradox/directives/repositories.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Library repositories
+--------------------
+
+The `@@repository` block is used to show example code for how to configure a
+library repository in a build tool, such as sbt.
+
+```markdown
+@@dependency[sbt,Maven,Gradle] {
```
Comment (pvlugter) [inline docs/src/main/paradox/directives/repositories.md:8]:
```suggestion
@@repository[sbt,Maven,Gradle] {
```

File: docs/src/main/paradox/directives/repositories.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  id="id1"
```
Comment (pvlugter) [inline docs/src/main/paradox/directives/repositories.md:9]: Maybe a full example id, like `"company-repository"` or something?

**Review by pvlugter — APPROVED** [review]
LGTM

Just that the example code is using `@@dependency`.

**Comment by ennru** [comment]
Now even tested on an existing site.

**Pull request merged**